_posts/2023-12-06-where-is-Strimzi-1.0.0.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title:  "Where is Strimzi 1.0.0?"
+date: 2023-12-06
+author: jakub_scholz
+---
+
+If you have been using Strimzi for some time, you might be wondering about its versioning.
+Why does Strimzi still have only some _zero-dot-something_ versions?
+Where is Strimzi 1.0.0?
+So in this blog post, we will look at the history behind it and our latest plans for the 1.0.0 release.
+
+_This blog post is a personal opinion of the author and might not represent the view of the whole Strimzi community._
+
+<!--more-->
+
+### The history
+
+Strimzi is almost 6 years old.
+For a software project, 6 years is not a _short time_.
+When we initially started it, we started it _boldly_.
+Our first version was not called 0.0.1 but 0.1.0.
+The 0.1.0 release was not yet based on the operator pattern.
+That came only with the 0.2.0 release.
+0.1.0 was just a bunch of YAML files to make it easier to deploy Apache Kafka.
+
+Neither 0.1.0 nor 0.2.0 were of course production-ready.
+They were just the first steps on a long road.
+But with every release, we added more and more features, fixed bugs, and improved things ... and suddenly, we had our first production users and the first companies started to offer commercial products based on Strimzi.
+With more and more users, we started thinking about releasing Strimzi 1.0.0.
+
+But there was a thing on the horizon ... the _ZooKeeper-less Apache Kafka_ (also called _KRaft mode_).
```

Comment:
Should we also mention that Kubernetes itself was evolving, and some core APIs were not at v1 stage (CRD for example)?